sql/_05_plcsql/_01_testspec/_02_declaration/_02_variable/_04_deeping/_03_varchar/cases/01_varchar_basic.sql

```diff
@@ -0,0 +1,50 @@
+--+ server-message on
+-- Verified for CBRD-24901
+
+create or replace procedure varchar_test1(t_query varchar) as
+	varchar_1 varchar(20);
+begin
+	execute immediate t_query into varchar_1;
+	
+	dbms_output.put_line('check size 20 (not default size)');
+	dbms_output.put_line('varchar: "' || varchar_1 ||'"');
+end;
+
+call varchar_test1('select cast(''abc'' as varchar(3)) from dual;');
+
+
+create or replace procedure varchar_test2() as
+	num_1 varchar(20) := 3;
+	num_2 varchar(5) := 8;
+begin
```

Comment:
num_1 varchar(20) := 2;
	num_2 varchar(5) := 10;

Reply:
num_1, num_2 is varchar type.
this case is not auto-cast by integer.
Therefore I add a num_1 > num_2 and num_2 > num_1 on this test case.